tests/test_mem_eff_attention.py

```diff
@@ -1750,6 +2174,7 @@ def test_f16_biasf32(self) -> None:
         with pytest.raises((ValueError, RuntimeError)):
             fmha.memory_efficient_attention(q, k, v, attn_bias=bias)
 
+    @disable_on_rocm
```

Comment:
```
`ck_decoderF` is not supported because:
    attn_bias type is <class 'torch.Tensor'>
`tritonflashattF` is not supported because:
    Embed dim 16 not supported
    dtype=torch.float32 (supported: {torch.bfloat16, torch.float16})
    attn_bias type is <class 'torch.Tensor'>
`ckF` is not supported because:
    dtype=torch.float32 (supported: {torch.bfloat16, torch.float16})
```